lib/Backend/InlineeFrameInfo.h

```diff
@@ -14,6 +14,7 @@ struct BailoutConstantValue {
     };
     void InitVarConstValue(Js::Var value);
     void InitFloatConstValue(FloatConstType value) { this->type = TyFloat64; this->u.floatConst.value = value; }
+    bool operator==(const BailoutConstantValue & bailoutConstValue);
```

Comment:
I don't like operator overloading in this way as I hides the true cost and meaning of the operation.  An IsEqual() function accomplishes the same but I can easy browse to the definition now to see what the code is actually doing.